core/sql/src/main/java/org/trafodion/sql/HBaseClient.java

```diff
@@ -648,6 +648,8 @@ public boolean truncate(String tblName, boolean preserveSplits, long transID)
            }
            else {
               TableName tableName = TableName.valueOf(tblName);
+              if (admin.isTableEnabled(tableName))
+                  admin.disableTable(tableName);
```

Comment:
Thanks @anoopsharma00.  I had used  truncate to delete the data in "_DTM_".TDDL hbase table.  It looks like the errors were ignored in that case and hence it didn't detect this requirement